src/__stories__/config.js

```diff
@@ -0,0 +1,8 @@
+const { configure } = require("@kadira/storybook")
+
+function loadStories() {
+  require("../components/__stories__/artwork")
+  require("../components/__stories__/artwork_grid")
+}
+
+configure(loadStories, module);
```

Comment:
Ah yes, I had tried globbing a while ago, but the issue turned out to be that this file actually only gets eval’ed in the browser, not in the Node.js process, and thus globbing doesn’t actually work/make sense.

However… there might be a way to do it by using the FuseBox in-browser API 🤔